rpm/toggle-monitor/toggle-monitor.spec

```diff
@@ -0,0 +1,24 @@
+Name:     toggle-monitor
+Version:  1
+Release:  1
+Summary:  Script to turn monitor on and off
+License:  Public Domain
+
+Requires: xrandr
+
+SOURCE0:  monitor-toggle
+SOURCE1:  10-keybindings-toggle-monitor
+
+%description
+RPM package with toggle-monitor script
+
+%install
```

Comment:
Please see this file:
https://github.com/vojtapolasek/vojtux/blob/master/specs/vojtux-docs-en.spec
I think the %install section could be empty.

Reply:
I think this can be the next step once we get the file in the repo and build new release.